packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/sqm-tax-document-step-1.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+Feature: Tax Form Step One
```

Comment:
I know these are just the specs for step 1 at the moment, but since the partner creation doesn't happen until the end of step 2. The specs should probably be put together because there are some end-to-end cases that need to be documented. 

For example, your spec "A user fills out Tax Form Step One with the required information" would provide a lot of value if it included step 2 and explicitly said that the partner was created. This way its clear when these things are happening and we can test for them in QA as well

Reply:
Its easy forget the specs that tie our subcomponents together in the user experience but sometimes they are the most important ones